OrbitCore/Profiling.h

```diff
@@ -30,7 +30,8 @@ inline TickType OrbitTicks(uint32_t a_Clock = 1 /*CLOCK_MONOTONIC*/) {
   return 1000000000ll * ts.tv_sec + ts.tv_nsec;
 }
 
-inline uint64_t NanoSecondsFromTicks(TickType start, TickType end) {
+// TODO: use similar naming for other functions
```

Comment:
Maybe it can just be done? It's only two functions.

Reply:
It's done